bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -560,8 +560,8 @@ CGINS-CTDBPC-50153,A01758,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C,Sea
 CGINS-CTDBPC-50154,A01759,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C,Sea-Bird Electronics,SBE 16plus V2,16-50154,,20150831,13535,,
 CGINS-CTDBPC-50187,,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50187,,,,,
 CGINS-CTDBPC-50188,,Sensor,0,CTD Pumped: CTDBP Series C,CTDBP SERIES C (C-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-50188,,,,,
-CGINS-CTDBPD-07209,A00083,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-7209,2.5,20121102,11150,,
-CGINS-CTDBPD-07239,A00132,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16-7239,2.5,20130108,11150,,
+CGINS-CTDBPD-07209,A00083,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16P71174-7209,2.5,20121102,11150,,
+CGINS-CTDBPD-07239,A00132,Sensor,0,CTD Pumped: CTDBP Series D,CTDBP SERIES D (D-16PLUSV2),Sea-Bird Electronics,SBE 16plus V2,16P71174-7239,2.5,20130108,11150,,

```

Comment:
Per WHOI_Asset_Tracking, I think this one should be 16P71879-7239.

Reply:
fixed